lib/floating_action_bubble.dart

```diff
@@ -3,11 +3,11 @@ import 'package:flutter/material.dart';
 class Bubble {
   const Bubble(
       {@required this.title,
-      @required this.titleStyle,
-      @required this.iconColor,
-      @required this.bubbleColor,
-      @required this.icon,
-      @required this.onPress});
+        @required this.titleStyle,
+        @required this.iconColor,
```

Comment:
don't mind my formatting

Reply:
This is good formatting sir . will merge this. thank you